polaris-core/src/main/java/org/apache/polaris/core/persistence/LocalPolarisMetaStoreManagerFactory.java

```diff
@@ -157,12 +157,7 @@ public synchronized StorageCredentialCache getOrCreateStorageCredentialCache(
     return storageCredentialCacheMap.get(realmContext.getRealmIdentifier());
   }
 
-  @Override
-  public void setMetricRegistry(PolarisMetricRegistry metricRegistry) {
-    // no-op
-  }
-
-  @Override
+  @Inject
```

Comment:
Why not just add `@Inject` to the field?

Reply:
can do